fail2ban/server/filter.py

```diff
@@ -354,34 +375,22 @@ def getIgnoreIP(self):
 	#
 	# Check if the given IP address matches an IP address/DNS or a CIDR
 	# mask in the ignore list.
-	# @param ip IP address
+	# @param ip IP address object
 	# @return True if IP address is in ignore list
 
 	def inIgnoreIPList(self, ip, log_ignore=False):
```

Comment:
Yes as with this PR, an IP address is being treated as a IPAddr object, the tests need to honor this change/fact. Unfortunately I don't know/understand what you mean with `asanyipaddr` or `asanyarray`. Do you have any reference to what you mean by that? If anything can be simplified, why not?